### ISSUE TITLE
Unable to add custom network request headers

### ISSUE THREAD

**Issue by sbatezat** [Countly/countly-sdk-flutter-bridge#282]
On Android, I'm able to addCustomNetworkRequestHeaders() (see https://support.countly.com/hc/en-us/articles/360037754031-Android#h_01HAVQDM5W8GFBCZ3B8CFT5BHF)

~~I do not know if it's available on iOS, but I need it.~~
On iOS, I'm also able to do it through _URLSessionConfiguration_
Example : 
```swift
// Add custom network request header
let session = URLSessionConfiguration.default
let customHeader = ["KEY": "value"]
session.httpAdditionalHeaders = customHeader
config.urlSessionConfiguration = session
```

On Flutter, there is no such method (not found on https://support.countly.com/hc/en-us/articles/360037944212-Flutter#h_01H930GAQ8DQ2KJTQCAFZTRHD9)

**Comment by turtledreams** [comment]
Hi @sbatezat, yes we don't give that option in Flutter yet. We will try to add it to next release if there won't be any blockers. Thank you for bringing it up.

**Comment by sbatezat** [comment]
fix in 24.11.0
